src/twisted/python/compat.py

```diff
@@ -150,6 +151,22 @@
 )
 
 
+def md5(data=b"", **kwargs):
+    """
+    Wrapper around hashlib.md5
+    Attempt call with 'usedforsecurity=False' if we get a ValueError, which happens when
+    OpenSSL FIPS mode is enabled:
+    ValueError: error:060800A3:digital envelope routines:EVP_DigestInit_ex:disabled for fips
+    """
+
+    try:
+        return hashlib.md5(data, **kwargs)
+    # the below value error will be hit only when fips is enabled
+    # so skip it while testing for code coverage
+    except ValueError:  # pragma: no cover
```

Comment:
`# pragma: no cover` should only be used for code that is not exected to be executed in production.

This branch, looks like normal business  for a FIPS library with FIPS mode disabled.
We need to have tests (I think we already done) and report the coverage for it.
I left a suggestion in terms of coverage